chromadb/test/db/test_system.py

```diff
@@ -256,7 +256,7 @@ def test_get_or_create_collection(sysdb: SysDB) -> None:
     result, created = sysdb.create_collection(
         name=collection.name,
         id=uuid.uuid4(),
-        configuration=CollectionConfiguration(),
+        configuration=CollectionConfigurationInternal(),
```

Comment:
trying to understand the implementation, could these be `CollectionConfiguration` instead? does a user ever need to touch `CollectionConfigurationInternal`?

Reply:
Users of Chroma should never touch it, it's a representation we use internally. Note that this test is for `sysdb.create_collection` which is internal to us.